Documents/cluster.md

```diff
@@ -3,4 +3,28 @@
 ## Cluster Management
 
 The project uses the [Akka Cluster HTTP Management](http://developer.lightbend.com/docs/akka-management/current/cluster-http-management.html) module to report on cluster member status as well as to perform cluster
-management tasks. Please refer to the documentation of this module for cluster management tasks (i.e. downing a node from the cluster)
\ No newline at end of file
+management tasks. Please refer to the documentation of this module for cluster management tasks (i.e. downing a node from the cluster)
+
+## Roles
+
+There are four different cluster roles, each managed within the cluster project.
+
+### API
+
+The API cluster role is responsible for the four different APIs: Admin, Public, Filing, and the TCP Panel Loader
+
+### Persistence
+
+The persistence cluster role contains actors responsible for tracking and persisting information about submissions, institutions, and LAR validation
+
+### Query
+
+This cluster role contains actors responsible for interacting with the backend database (currently Cassandra)
+
+### Publication
+
+The publication role is responsible for creating and publishing the A&D reports from submitted LAR data
```

Comment:
I would change to something like this: "The publication role is responsible for generating publication artifacts based on HMDA data (i.e. A&D reports, Modified LAR, etc.)"